src/pages/react/Installations/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: React App Dependencies Installation 
+---
+## Pre-requisites 
+We need the right combinations of `Babel` ,` Webpack` and `NPM `dependencies to get started with a basic React Application.  `create-react-app` a command line tool is provided by React developers to abstract all the complexities involved in setting up a react project.
+What goes under the hood when we use this command is what we will be explaining here.
+
+```C
+1.	Create an empty directory and change the directory .
+2.	Run `npm init` command that will ask  a bunch of questions, and then write a package.json for us.
+3.	Now run the npm command to install `react` and `react-dom` which will be stored in `node_modules` folder.
+      
+```
+Here are the terminal commmands:-
+
+```C
```

Comment:
This should be in a code block with `shell` formatting.

Please change this to ` ```shell` or ` ```sh`